src/containers/sign-on/utils/formatSocialProfile.ts

```diff
@@ -4,6 +4,7 @@ import { resizeImage } from 'utils/imageProcessingUtil'
 const GENERAL_ADMISSION = process.env.REACT_APP_GENERAL_ADMISSION
 
 export const MAX_HANDLE_LENGTH = 16
+export const MAX_DISPLAY_NAME_LENGTH = 32
```

Comment:
can we share this const?
https://github.com/AudiusProject/audius-client/blob/7a4a87b0cc6d89cb6d5478223dc421e609b4542e/src/containers/sign-on/components/ProfileForm.js#L88